src/cell.js

```diff
@@ -215,9 +215,18 @@ class Cell {
   wrapWithStyleColors(styleProperty, content) {
     if (this[styleProperty] && this[styleProperty].length) {
       try {
-        let colors = require('@colors/colors/safe');
+        let colors = require('ansis');
         for (let i = this[styleProperty].length - 1; i >= 0; i--) {
-          colors = colors[this[styleProperty][i]];
+          let style = this[styleProperty][i];
+          let isHex = style.startsWith('hex');
```

Comment:
Add support for new truecolor syntax using hex code, e.g.: `hex(#FFD700)` and `bgHex(#FFD700)`.